css/index.css

```diff
@@ -1,3 +1,328 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+.container {
+    width: 880px;
+    margin: auto;
+}
 
+@media(max-width: 800px) {
+    .container {
+        width: 100%;
+    }
+}
 
+/* Header Styles */
+
+header {
+    width: 100%;
+    display: flex;
+}
+
+@media(max-width: 800px) {
+    header {
+        flex-wrap: wrap;
+    }
+}
+
+@media(max-width: 500px) {
+    header {
+        justify-content: center;
+    }
+}
+
+header nav {
+    margin-top: 40px; 
+    width: 610px;
+    justify-content: space-between;
+    display: flex;
+}
+
+@media(max-width: 800px) {
+    header nav {
+        width: 768px;
+        padding: 0 30px;
+
+    }
+}
+
+@media(max-width: 500px) {
+    header nav {
+        width: 100%;
+        flex-direction: column;
+        margin: 0;
+        padding: 0;
+    }
+}
+
+header nav a {
+    text-decoration: none;
+    color: #b0b0b0;
+}
+
+@media(max-width: 500px) {
+    header nav a {
+        width: 100%;
+        display: block;
+        text-align: center;
+        padding: 20px 0;
+        font-size: 20px;
+        border-bottom: 1px solid gray;
+    }
+}
+
+header nav a:hover {
+    text-decoration: underline;
+    color: black;
+}
+
+@media(max-width: 500px) {
+    header nav a:hover {
+        background: gray;
+    }
+}
+
+header .logo {
+    margin: 28px 0 0 91px;
+}
+
+@media(max-width: 500px) {
+    header .logo {
+        margin: 20px 0;
+    }
+}
+
+@media(max-width: 800px) {
+    header .logo {
+        order: -1;
+        display: block;
+        margin: 30px auto 0;
+    }
+}
+
+/* Call To Action (CTA) Styles */
+
+.cta {
+    display: flex;
+    margin-top: 72px;
+    justify-content: flex-end;
+}
+
+@media(max-width: 800px) {
+    .cta {
+        justify-content: center;
+    }
+}
+
+@media(max-width: 500px) {
+    .cta {
+        margin-top: 50px;
+        flex-flow: column;
+        align-items: center
+    }
+}
+
+.cta .cta-text {
+    font-size: 72px;
+    text-align: center;
+    letter-spacing: 5px;
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+    align-items: center;
+    margin: 0 82px 0 0;
+}
+
+@media(max-width: 800px) {
+    .cta .cta-text {
+        margin: 0;
+    }
```

Comment:
you'll want to add in some media queries here for the h1 text to scale it down as the viewport size changes